src/App.tsx

```diff
@@ -1,14 +1,64 @@
-import React from 'react';
+import React, {
+  useCallback, useMemo, useEffect, useState,
+} from 'react';
+import debounce from 'lodash.debounce';
```

Comment:
You have to add all files with changes to git and push them (including package.json):
![image](https://github.com/mate-academy/react_autocomplete/assets/86923089/ebf8587d-8737-488c-954d-3dc0249eff09)